### DISCUSSION TITLE
Kubernetes Error / Rancher crashing after admin prvileges revoked by IT security software

### DISCUSSION THREAD

**Discussion by ryan-harness** [rancher-sandbox/rancher-desktop#6474] [Q&A]
Getting a Kubernetes error and Rancher is crashing because admin privileges were removed from computers by IT security software. Looks like Rancher expects this user to be part of the Sudoer’s memory (which refreshes on restart). Has anyone experienced any issues with Privilege Access Management on their computers and if so, what was the fix? The Software basically removes the user from the sudoer's file and the user must elevate their rights before getting access. This still doesn't fix the issue however. Anyone encounter anything like this before and if so, what was the fix or workaround?

<img width="1132" alt="Screenshot 2024-02-13 at 10 53 13 AM" src="https://github.com/rancher-sandbox/rancher-desktop/assets/159948261/baf4ce3a-2a3e-42ee-ad57-9f9ded8d5702">

**Comment by jandubois** [comment]
When Rancher Desktop runs with admin mode, then it writes `/etc/sudoers.d/zzzzz-rancher-desktop-lima` to allow it to run a specific list of commands without requiring the `sudo` password. These commands are needed to start the VMNET networking daemons.

If this mechanism doesn't work for configuration; then you probably need to run without admin mode. It just means you no longer get a routable IP address but need to rely on port forwarding. And if you are using moby, then the `docker.sock` cannot be created in `/var/run` but in your home directory. Most client software shouldn't care because we set up a docker context that points to the socket location.

On the bright side, running without admin mode means you no longer need to enter your password after each reboot because macOS wipes `/var/run` on boot.

**Reply by ryan-harness** [reply]
Hi @jandubois thank you for your reply. Regarding admin mode are you referring to from within Rancher or admin from the laptop/machine itself? I'm trying to see if the security softweare can be configured in a way as such, or maybe some kind of whitelist can be performed on their end.

**Reply by jandubois** [reply]
I'm talking about disabling admin mode in the application. It will then no longer require to run `sudo` commands, with the functionality I listed above disabled.
